aks/provider.tf

```diff
@@ -5,6 +5,15 @@ provider "azurerm" {
   version = "~> 1.27"
 }
 
+provider "kubernetes" {
+  load_config_file       = false
+  host                   = data.aws_eks_cluster.cluster.endpoint
+  cluster_ca_certificate = base64decode(data.aws_eks_cluster.cluster.certificate_authority.0.data)
+  token                  = data.aws_eks_cluster_auth.cluster.token
+  version                = "1.10.0"
+  alias                  = "override"
+}
+
```

Comment:
I get the version pinning to 1.10.0 because of the failure to initialize bug in 1.11 but why are you using aws_eks_cluster variable name for AKS?  It should be derived from the appropriate azurerm cluster resource as defined in main.tf:, e.g.

`host                   = "${azurerm_kubernetes_cluster.k8s.kube_config.0.host}" `

Reply:
You are right, thanks for the review!

Instead of adding a resource override, I have just changed the versions of the kubernetes resources, which seems the correct approach.